src/main/kotlin/org/jitsi/nlj/transform/node/incoming/DuplicateTermination.kt

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright @ 2018 - Present, 8x8 Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jitsi.nlj.transform.node.incoming
+
+import java.util.TreeMap
+import org.jitsi.nlj.PacketInfo
+import org.jitsi.nlj.stats.NodeStatsBlock
+import org.jitsi.nlj.transform.node.TransformerNode
+import org.jitsi.rtp.rtp.RtpPacket
+import org.jitsi.utils.LRUCache
+import org.jitsi.utils.logging2.Logger
+
+/**
+ * A node which drops packets with SSRC and sequence number pairs identical to ones
+ * that have previously been seen.
+ *
+ * (Since SRTP also has anti-replay protection, the normal case where duplicates
+ * will occur is after the [RtxHandler].)
```

Comment:
nit: maybe rephrase as:
```
(Since SRTP also has anti-replay protection, the normal case where duplicates will occur is after the [RtxHandler], since duplicate packets are sent over RTX for probing.)
```
?

I think I'm just looking for a way to tie it to what is it that happens within `RtxHandler` that causes the dupes, rather than just stating it happens after?  I dunno. A thought.